includes/sanitizers/class-amp-base-sanitizer.php

```diff
@@ -440,14 +440,15 @@ protected function has_dev_mode_exemption( DOMNode $node ) {
 	/**
 	 * Check whether a certain node should be exempt from validation.
 	 *
-	 * @deprecated Use AmpProject\DevMode::isExemptFromValidation( $node ) instead.
+	 * @since 1.5 Exempts DOMNode if the 'data-ampdevmode' attribute has the value '_amp_exempt'.
 	 *
 	 * @param DOMNode $node Node to check.
 	 * @return bool Whether the node should be exempt from validation.
 	 */
 	protected function is_exempt_from_validation( DOMNode $node ) {
-		_deprecated_function( 'AMP_Base_Sanitizer::is_exempt_from_validation', '1.5', 'AmpProject\DevMode::isExemptFromValidation' );
-		return DevMode::isExemptFromValidation( $node );
+		return DevMode::isExemptFromValidation( $node )
+			&& $node instanceof DOMElement
+			&& '_amp_exempt' !== $node->getAttribute( AMP_Rule_Spec::DEV_MODE_ATTRIBUTE );
```

Comment:
I think this can be reverted based on my comment on the `sanitize` method below.